lib/nacre/journal/entry.rb

```diff
@@ -1,3 +1,5 @@
+require 'bigdecimal'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.